content/cpp/concepts/vectors/vectors.md

```diff
@@ -56,6 +56,24 @@ std::cout << order[0];
 std::cout << order[2];
 ```
 
+A safer way of accessing elements in a vector can be achieved by using the `.at()` function.
+This function performs bounds checking on the vector and will throw an error if you are trying to access an element that is out of bounds.
+
+```cpp
+// First element
+std::cout << order.at(0);
+
+// Last element
+std::cout << order.at(2);
+
+// Out of bounds
+std::cout << order.at(100);
+
+// terminate called after throwing an instance of 'std::out_of_range'
+// what():  vector::_M_range_check: __n (which is 100) >= this->size() (which is 3)
+// Aborted (core dumped)
+```
+
```

Comment:
```suggestion

The code above will print the following error:

```

Reply:
And then let's move the comments on lines # 72 - 74 to their own \```shell snippet, like this:

```shell
terminate called after throwing an instance of 'std::out_of_range'
  what():  vector::_M_range_check: __n (which is 100) >= this->size() (which is 3)
Aborted (core dumped)
```